packages/form-js-viewer/test/spec/Form.spec.js

```diff
@@ -474,6 +476,100 @@ describe('Form', function() {
   });
 
 
+  describe('focus and blur events should trigger for', async function() {
+
+    const runFocusBlurTest = async function(id, index, selector) {
```

Comment:
I like this! To even save more duplication, we can do something[ like in other places](https://github.com/bpmn-io/form-js/blob/develop/packages/form-js-editor/test/spec/FormEditor.spec.js#L1342)

```js
const runFocusBlurTest = function(id, index, selector) {
  it('focus and blur events should trigger for ' + id, async function() {
    
    // given
    // ...
  });
}

runFocusBlurTest('number', 0, 'input');

runFocusBlurTest('date', 1, 'input');
```